packages/aragon-wrapper/src/cache/index.js

```diff
@@ -17,6 +17,52 @@ import * as configurationKeys from '../configuration/keys'
  *        Require the cache to downgrade to localstorage even if IndexedDB is available
  */
 export default class Cache {
+  static cacheTracker
+
+  static initCacheTracker = async () => {
+    if(!Cache.cacheTracker) {
```

Comment:
This singleton implementation could get improved with a ``Cache.getTracker()``. Such a method would remove some code duplications here in this class.